src/app/inventory/ItemPopupTrigger.tsx

```diff
@@ -13,6 +13,7 @@ let otherDialog: any = null;
 interface Props {
   item: DimItem;
   children?: React.ReactNode;
+  onShiftClicked?(item: DimItem): void;
```

Comment:
If you add this to `InventoryItem` then I think you can just handle the event (return false) in before it bubbles up to the `ItemPopupTrigger`, if you choose to provide a handler for it.